frontend/src/pages/PipelineDetails.tsx

```diff
@@ -305,15 +334,10 @@ class PipelineDetails extends Page<{}, PipelineDetailsState> {
         versions,
       });
     } else {
-      const graph = await this._createGraph(templateString);
-      let reducedGraph = graph ? transitiveReduction(graph) : undefined;
-      if (graph && reducedGraph && compareGraphEdges(graph, reducedGraph)) {
-        reducedGraph = undefined; // disable reduction switch
-      }
       this.setStateSafe({
         graph,
         reducedGraph,
-        graphV2: '',
+        graphV2: undefined,
```

Comment:
nit: omit graphV2? since it's already undefined

Reply:
That is right, but I set graphV2 to explicitly control the visibility of v2 pipeline detail page during rendering. For example, during each load(), if I get a v1 pipeline, I will erase the `graphV2` content, and only show `graph`. So `render()` will switch to use v1 pipeline detail page.

Reply:
I see, makes sense to me

Reply:
Thanks for confirming!